modules/common/util/util.h

```diff
@@ -36,6 +36,8 @@
 #include "modules/common/proto/pnc_point.pb.h"
 
 #include "modules/common/math/vec2d.h"
+#include "cyber/common/types.h"
+#include "cyber/common/log.h"
```

Comment:
Make headers in order.

Reply:
Thanks xiangquan. I will fix this issue:)